src/App.css

```diff
@@ -2351,21 +2351,26 @@ code, .code {
 /* column 1 */
 .table-view > .children > .child > .thought-container {
   display: table-cell;
-  text-align: right;
+  /*text-align: right;*/
```

Comment:
> Pull Requests that solve the given issue but introduce a regression (that is, break or remove functionality) are not considered solutions and will not be accepted. Preserving existing functionality is an implicit requirement.

https://github.com/cybersemics/em/blob/dev/CONTRIBUTING.md#regressions

Reply:
with the `text-align: right` the cursor will be too far from the text, I will try and explore other options though and see how they will work.

Reply:
Yes, you will need to find a solution that still allows the text to be right-aligned.